packages/devtools-launchpad/src/components/LandingPage.css

```diff
@@ -93,6 +93,10 @@
   transition: var(--base-transition);
 }
 
+.landing-page .panel header input[type=button] {
+  flex: 0.25;
```

Comment:
I'm not sure about this, maybe we could only have some padding on the button and let the search input take all the remaining space ?
What do you think about it ?